### PR TITLE
add oereb lex model creation method for more convenient use

### PR DESCRIPTION
**Pull request by vvmruder**

### CODE REVIEW

File: pyramid_oereb/contrib/scripts/__init__.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import optparse
```
Comment (sbrunner) [inline pyramid_oereb/contrib/scripts/__init__.py]: optparse is deprecated, you should use argparse...
see: https://docs.python.org/3/library/optparse.html

**Comment by vvmruder** [comment]
I will add it when merged in the documentation branch.

**Comment by vvmruder** [comment]
@sbrunner: I think we use this optparse for all console methods. Shouldn't we do an allround refactoring in a separate branch to clean this?

**Comment by sbrunner** [comment]
> Shouldn't we do an allround refactoring in a separate branch to clean this?

Looks good :-)

**Pull request merged**